flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobServerDeleteTest.java

```diff
@@ -92,6 +99,69 @@ public void testDeleteSingle() {
 			catch (IllegalStateException e) {
 				// expected
 			}
+
+			// delete a file directly on the server
+			server.delete(key2);
+			try {
+				server.getURL(key2);
+				fail("BLOB should have been deleted");
+			}
+			catch (IOException e) {
+				// expected
+			}
+		}
+		catch (Exception e) {
+			e.printStackTrace();
+			fail(e.getMessage());
+		}
```

Comment:
Absolutely right - I was told this was an issue some versions ago with JUnit or so... was just copying the code from the other tests in this class. Let me create a cleanup-PR on top of the newest PR in this series, i.e. #4238